README.md

```diff
@@ -85,6 +85,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Lime | Optional | Restricted | Restricted | Restricted | 2020-03-09 |
 | Lyft | Encouraged | Non-critical restricted | ? | ? | 2020-03-06 |
 | LeaseCloud | Required | Restricted | Restricted | Restricted | 2020-03-11 |
+| mabl | Strongly encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```

Comment:
Do you have a public-facing link to this post? It's not required by any means, but it's just a quick check on our end.